fastlane/lib/fastlane/actions/latest_testflight_build_number.rb

```diff
@@ -72,7 +78,7 @@ def self.available_options
                                        description: "The version number whose latest build number we want",
                                        optional: true),
           FastlaneCore::ConfigItem.new(key: :initial_build_number,
-                                       env_name: "INTITIAL_BUILD_NUMBER",
+                                       env_name: "INITIAL_BUILD_NUMBER",
```

Comment:
While this is okay, we can't just change the existing parameter, as it would break existing setups. Please just add a `default_value` with `ENV["INITIAL_BUILD_NUMBER"]` so that we support both spellings.